packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -89,140 +103,131 @@ const onKeyDown: React.KeyboardEventHandler = e => {
   }
 };
 
-function MongoSelect(
-  props: ProjectMongoSelectProps | OrganizationMongoSelectProps,
-) {
-  const {
-    onClick,
-    variant,
-    onChange,
-    className,
-    urls,
-    isActive = false,
-  } = props;
-
-  let trigger, footer;
-  const { mongoSelect } = urls;
-
-  if (isProject(props)) {
-    trigger = <ProjectTrigger current={props.current} className={className} />;
-    footer = (
-      <li onKeyDown={onKeyDown} role="none" className={projectButtonStyle}>
-        <FocusableMenuItem>
-          <Button href={mongoSelect.viewAllProjects} as="a">
-            View All Projects
-          </Button>
-        </FocusableMenuItem>
-        <FocusableMenuItem>
-          <Button href={mongoSelect.newProject} as="a">
-            + New Project
-          </Button>
-        </FocusableMenuItem>
-      </li>
-    );
-  } else {
-    trigger = (
-      <OrganizationTrigger
-        current={props.current}
-        className={className}
-        urls={urls}
-        isActive={isActive}
-      />
-    );
-    footer = (
-      <MenuItem onKeyDown={onKeyDown} href={mongoSelect?.viewAllOrganizations}>
-        <strong className={viewAllStyle}>View All Organizations</strong>
-      </MenuItem>
-    );
-  }
-
-  const renderProjectOption = (
-    props: ProjectMongoSelectProps,
-    datum: ProjectInterface,
-  ) => {
-    const content = (
-      <ProjectOption
-        projectName={datum.projectName}
-        href={props.constructProjectURL(datum.orgId, datum.projectId)}
-      />
-    );
+function OrgSelect({
+  current,
+  data,
+  className,
+  urls,
+  isActive,
+  onChange,
+  onClick,
+  constructOrganizationURL,
+}: OrganizationMongoSelectProps) {
+  const renderOrganizationOption = (datum: OrganizationInterface) => {
+    const { orgId, orgName, planType } = datum;
 
     return (
       <MenuItem
-        key={datum.projectId}
+        key={orgId}
         className={menuItemContainerStyle}
         onClick={onClick}
+        href={constructOrganizationURL(orgId)}
       >
-        {content}
+        <div className={orgOptionContainerStyle}>
+          <span className={nameStyle}>{orgName}</span>
+          <span className={productStyle}>{formatPlanType(planType)}</span>
+        </div>
       </MenuItem>
     );
   };
 
-  const renderOrganizationOption = (
-    props: OrganizationMongoSelectProps,
-    datum: OrganizationInterface,
-  ) => {
-    const content = (
-      <OrganizationOption
-        orgName={datum.orgName}
-        planType={datum.planType}
-        href={props.constructOrganizationURL(datum.orgId)}
-      />
-    );
+  const errorMessage =
```

Comment:
nit, this variable isn't used anywhere else. Better to inline IMO rather than having the reader read this variable here and then have to read where it's being used to understand the full behavior of the component.

Reply:
It's going to end up being more complicated logic down the line, because IIRC the error message will be different depending on what product you're in (CM, OM or Atlas) but can inline for now and can always pull out later if it becomes necessary